modules/exploits/multi/http/jenkins_script_console.rb

```diff
@@ -187,9 +161,19 @@ def exploit
         end
       else
         print_status('Logging in...')
+        # get that first cookie that's needed by newer versions
+        res = send_request_cgi({ 'uri' => normalize_uri(@uri.path, 'login'), 'keep_cookies' => true })
+        fail_with(Failure::UnexpectedReply, 'Unexpected reply from server') unless res&.code == 200
```

Comment:
Just looping in the suggestion that we could do error detection here when the user has pointed the module to a jenkins instance running on `/` and hasn't updated the TARGETURI value from the default `/jenkins` to `/`

https://github.com/rapid7/metasploit-framework/pull/17013#issuecomment-1248099303